dbfit-java/informix/src/main/java/dbfit/environment/InformixEnvironment.java

```diff
@@ -0,0 +1,309 @@
+package dbfit.environment;
+
+import dbfit.annotations.DatabaseEnvironment;
+import dbfit.api.AbstractDbEnvironment;
+import dbfit.fixture.StatementExecution;
+import dbfit.util.DbParameterAccessor;
+import dbfit.util.Direction;
+import dbfit.util.NameNormaliser;
+
+import java.math.BigDecimal;
+import java.sql.*;
+import java.util.*;
+import java.util.regex.Pattern;
+
+import static dbfit.util.Direction.*;
+import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+
+@DatabaseEnvironment(name="Informix", driver="com.informix.jdbc.IfxDriver")
+public class InformixEnvironment extends AbstractDbEnvironment {
+
+    public InformixEnvironment(String driverClassName) {
+        super(driverClassName);
+        dbfitToJdbcTransformerFactory.setTransformer(dbfit.util.NormalisedBigDecimal.class, new BigDecimalExactClassTransformer());
+    }
+
+    protected String parseCommandText(String commandText) {
+        commandText = commandText.replaceAll(paramNamePattern, "?");
+        return super.parseCommandText(commandText);
+    }
+
+    private static String paramNamePattern = "[@:]([A-Za-z0-9_]+)";
```

Comment:
Would you suggest laying out all of the attributes and then all of the methods, or keeping the attributes just before the method that uses them (if only one)?